samples/bluetooth/mesh/prj_bbc_microbit.conf

```diff
@@ -2,8 +2,17 @@ CONFIG_INIT_STACKS=y
 CONFIG_MAIN_STACK_SIZE=512
 CONFIG_MICROBIT_DISPLAY=y
 CONFIG_SYS_CLOCK_TICKS_PER_SEC=250
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
 CONFIG_GPIO=y
 
+CONFIG_BT_SETTINGS=y
+CONFIG_FLASH=y
+CONFIG_FLASH_PAGE_LAYOUT=y
+CONFIG_FLASH_MAP=y
+CONFIG_FCB=y
+CONFIG_SETTINGS=y
+CONFIG_SETTINGS_FCB=y
```

Comment:
Sorry if I missed this, is it documented somewhere that the sample now uses FCB?

Reply:
So how does this work then? persistent storage is only enabled when there's no GATT enabled?

Reply:
The GATT possibility for this sample app is a special-case that you have to explicitly choose with `CONF_FILE=microbit_gatt.conf`. If you don't do that you just get a "normal" mesh app without PB-GATT or GATT Proxy features.

Reply:
OK thanks for the explanation.